pkg/virt-controller/watch/export/export.go

```diff
@@ -1027,10 +1027,20 @@ func (ctrl *VMExportController) findCertByHostName(hostName string, certs []*x50
 	return "", nil
 }
 
-func buildPemFromCert(cert *x509.Certificate) string {
+func (ctrl *VMExportController) buildPemFromCert(matchingCert *x509.Certificate, allCerts []*x509.Certificate) (string, error) {
```

Comment:
how does this code handle the case when there may be multiple entries for the same hostname?  This will happen when certs are rotated.  I think it will just return the first encountered?  But what if that's an old one that will expire soon?  Will the export get updated?  I know these questions not entirely related to changes in this PR.

Reply:
Yeah definitely don't take rotation into account currently. I suppose the bundle could contain multiple certs with the same name. I guess if multiple are valid, I should return the one with the expire that is furthest into the future?

Reply:
Sure, but do we have to watch whatever source we got the cert from so that we catch updates/rotation?